src/sentry/search/events/filter.py

```diff
@@ -926,11 +1006,7 @@ def _environment_filter_converter(
         else:
             return env_conditions[0]
 
-    def _project_slug_filter_converter(
-        self,
-        search_filter: SearchFilter,
-        _: str,
-    ) -> Optional[WhereType]:
+    def _project_slug_filter_converter(self, search_filter: SearchFilter) -> Optional[WhereType]:
```

Comment:
This method still has the side effect of updating `self.projects_to_filter`. Not seeing a clean way of removing this side effect yet, so going to leave it alone for now.